database.go

```diff
@@ -87,6 +102,39 @@ func (d *Database) GetDocument(ctx context.Context, id string) (Document, error)
 	return doc, err
 }
 
+func (d *Database) GetDocumentVersions(ctx context.Context, id string, withContent bool) ([]Document, error) {
```

Comment:
should be `GetDocumentVersions(ctx context.Context, documentID string, withContent bool)`